sar.drush.inc

```diff
@@ -269,6 +349,24 @@ function drush_sar_parse_options() {
   );
 }
 
+/**
+ * Private function to do a DB REPLACE query on the `js_injector_rule` table.
+ *
+ */
+function _drush_sar_replace_js_injector($options) {
+
+  // views_display.
+  $query = db_update("js_injector_rule")
+    ->expression("js", "REPLACE(js, :search, :replace)", array(':search' => $options['search'], ':replace' => $options['replace']));
+  if ($options['regex']) {
+    $query->condition("link_path", $options['regex'], "REGEXP");
+  }
+  $result_data = $query->execute();
+
+  echo "Replaced \"" . $options['search'] . "\"  with \"" . $options['replace'] . "\" in all JS Injector rules.";
```

Comment:
I just copypasta-ed the `drush sarm` code here. Any reason for using `echo` rather than `drush_log()`?

Reply:
I don't remember there being a reason to avoid drush_log. Probably just a lazy developer.